pybuildkite/builds.py

```diff
@@ -108,6 +111,10 @@ def list_all(
             "commit": commit,
             "page": page,
         }
+
+        if include_retried_jobs:
```

Comment:
Perhaps it would be cleaner to have an entry in the `query_params` dict just be `"include_retried_jobs=True if include_retried_jobs else None`. The `None` should get removed, thus removing the param.

Reply:
I was looking for something like this, thanks!

Reply:
I went for `"include_retried_jobs": True if include_retried_jobs is True else None` so that only `include_retried_jobs=True` sets the flag, not some arbitrary string like `include_retried_jobs='no'`. Also updated the docstring for `include_retried_jobs` a bit.

Reply:
Good call!